packages/mobile/ios/AudiusReactNative/Info.plist

```diff
@@ -89,6 +89,10 @@
   <true/>
   <key>LSSupportsOpeningDocumentsInPlace</key>
   <true/>
+	<key>LSApplicationQueriesSchemes</key>
+	<array>
+		<string>itms-apps</string>
```

Comment:
cool, just curious what this is?

Reply:
This is needed for linking out to the app store. Not entirely sure what it's for, but seems like it's allowing itms-apps links